### ISSUE TITLE
 Research `test_agent_groups`

### ISSUE THREAD

**Issue by Rebits** [wazuh/wazuh-qa#3795]
## Description

As part of #3776, in this issue, we will investigate the `test_agent_default_group_added` test in particular. We must verify that the test is stable and passes correctly.

**Comment by Rebits** [comment]
### Update 20/01/2023

```
FAILED system/test_cluster/test_agent_groups/test_agent_groups.py::test_agent_groups_create_remove_group - TimeoutError: Did not found the expected callback in wazuh-master: .*Master.*Local agent-groups.*Star...
========================================================================================== 1 failed in 824.56s (0:13:44) ==========================================================================================
```

**Comment by juliamagan** [comment]
## Update 25/01/2023

Gathered information on what the test does in order to have documentation.

**Comment by juliamagan** [comment]
## Update 27/01/2023

The fix of this test will be worked on in #3865, so we will close this issue.
